### PR TITLE
fix: replace method name with a generic one

### PR DESCRIPTION
**Pull request by AdriandeCita**
### Description

The section describes generic method interface, the use of `groupToMap` doesn't make any sense here.

### CODE REVIEW

File: files/en-us/web/javascript/reference/global_objects/array/index.md [hunk 1]
```diff
@@ -177,7 +177,7 @@ Where `callbackFn` takes three arguments:
 - `index`
   - : The index of the current element being processed in the array.
 - `array`
-  - : The array `groupToMap()` was called upon.
+  - : The array `method()` was called upon.
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/reference/global_objects/array/index.md:180]:
```suggestion
  - : The array that the method was called upon.
```

**Review by Josh-Cena — APPROVED** [review]
Great catch. Thank you!

**Comment by taciojoga10** [comment]
Great catch. Thank you! 

<a href=https://www.youtube.com/watch?v=Vgeo9DbI8xg>alpilean</a>

**Comment by taciojoga10** [comment]
<a href=https://www.youtube.com/watch?v=Vgeo9DbI8xg>alpilean</a>

**Pull request merged**